content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,81 @@ menu:
 weight: 101
 ---
 
+## v2.2 [TBD]
+
+### Features
+
+#### Replicate data remotely
+
+- Add the option to [replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+This feature helps you to collect data locally from multiple InfluxDB OSS edge deployments, analyze and aggregate the data, and then send the data to one or more buckets on a remote InfluxDB Cloud instance. If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data is sent when network connectivity is restored.
+
+#### Flux updates
+
+- Update to [Flux v0.150.0](/flux/v0.x/release-notes/#v01500-2022-01-19).
+- Option to log Flux queries that are cancelled because of server shutdown.
+
+#### Build maintenance
+
+- Upgrade `protobuf` library.
+- https://github.com/influxdata/influxdb/commit/5a1e375): Build: allow to build on FreeBSD
+
+#### Task metadata
+
+- (https://github.com/influxdata/influxdb/commit/504f0e4): Passing `type=basic` returns task metadata without query text.
+
+#### Metrics
+
+(https://github.com/influxdata/influxdb/commit/3a81166): Added metrics collection for replications
```

Comment:
@pierwill to be sure, did we add these replication metrics for troubleshooting into a task?: https://github.com/influxdata/docs-v2/issues/3459#issuecomment-988983567

Reply:
I think this is a little confusing.  This indicates that new metrics are now exposed which describe the details of the Durable Subscription -- how much data is being replicated, how much is enqueued, etc.  What this means is that if you use a scraper to gather metrics from InfluxDB OSS....there will be a new set of metrics related to this feature.  If you use Telegraf to gather metrics from InfluxDB OSS...there will be a new set of metrics related to this feature.

Reply:
> @pierwill to be sure, did we add these replication metrics for troubleshooting into a task?: [#3459 (comment)](https://github.com/influxdata/docs-v2/issues/3459#issuecomment-988983567)

Created a breakout issue: https://github.com/influxdata/docs-v2/issues/3769